src/cljs/rems/profile.cljs

```diff
@@ -100,7 +100,7 @@
        {:title (text :t.profile/your-details)
         :always [:<>
                  [user/username (:user identity)]
-                 [user/attributes (:user identity)]]}]]]))
+                 [user/attributes attributes false]]}]]]))
```

Comment:
Passing booleans always looks so opaque, but in this case I don't think we want full keyword args just for one boolean... Nothing to fix here, just thinking out loud.

Reply:
I also thought so, but figured this is not that big of a deal.